packages/share/android/src/main/java/io/flutter/plugins/share/MethodCallHandler.java

```diff
@@ -21,23 +21,27 @@
 
   @Override
   public void onMethodCall(MethodCall call, MethodChannel.Result result) {
+    String text, subject;
+    List<String> paths, mimeTypes;
```

Comment:
nit: move to `"shareFiles"` case, since that's the only place it's used. 

Also for these, please use one var per declaration, per https://google.github.io/styleguide/javaguide.html#s4.8.2-variable-declarations

Reply:
@dnfield PR updated . Thank you